detox/e2e/test/smoke_test/message_reply.e2e.js

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// *******************************************************************
+// - [#] indicates a test step (e.g. # Go to a screen)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element testID when selecting an element. Create one if none.
+// *******************************************************************
+
+import {
+    ChannelScreen,
+    ThreadScreen,
+} from '@support/ui/screen';
+import {
+    Channel,
+    Post,
+    Setup,
+} from '@support/server_api';
+
+describe('Message Reply', () => {
+    let testChannel;
+    let parentPost;
+    let parentPostItem;
+    let replyPost;
+    let replyPostItem;
+
+    beforeAll(async () => {
+        const {team, user} = await Setup.apiInit();
+
+        const {channel} = await Channel.apiGetChannelByName(team.name, 'town-square');
+        testChannel = channel;
+
+        // # Open channel screen
+        await ChannelScreen.open(user);
+    });
+
+    afterAll(async () => {
+        await ChannelScreen.logout();
```

Comment:
Test passed on Android but failed here due to keyboard covering the screen. Please check again.
<img width="779" alt="Screen Shot 2021-01-27 at 9 52 56 PM" src="https://user-images.githubusercontent.com/5334504/106000842-3b64af00-60ea-11eb-9f0b-dbb8b2ad2c63.png">

Reply:
@saturninoabril It doesn't happen for me locally but I've added a step so that keyboard is minimized. Please run again. Thanks!